exampleSite/content/docs/advanced/customization.md

```diff
@@ -62,6 +62,27 @@ You may add custom scripts to the end of the head for every page by adding the f
 layouts/partials/custom/head-end.html
 ```
 
+## Custom Footer Section
+
+You can add a custom section the footer between the language/theme buttons and the Copyright/Powered By section by creating a file `layouts/partials/custom/footer.html` in your site __and__ setting `customSection: true` in the `footer` section of your `hugo.yaml`:
+
+```yaml {filename="hugo.yaml"}
+ footer:
+    enable: true
+    customSection: true
```

Comment:
I would avoid adding a new flag if possible

Reply:
I can remove it but then I don't know when I should show the separator and when I shouldn't.

Reply:
Actually, I just remove the separator, people can add it back in their custom footer.